tests/unit_tests/libs/controllers/test_predictor.py

```diff
@@ -305,7 +305,7 @@ def test_explain_prediction(self):
         explanation_new = result[0].explanation['numeric_y']
         assert isinstance(explanation_new['predicted_value'], int)
         assert isinstance(explanation_new['confidence_interval'],list)
-        assert isinstance(explanation_new['confidence_interval'][0],int)
+        assert isinstance(explanation_new['confidence_interval'][0],float)
```

Comment:
Not sure where to leave this comment, but I fail to see where `confidence_interval` actually ends up being set by the NC model ?